public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -219,4 +220,20 @@ export class EhrViewComponent implements OnInit, OnDestroy {
       return this.sourcesChartHelpText;
     }
   }
+  public expandRow(concepts, r) {
+    const index = concepts.findIndex(x => x.conceptId === r.conceptId);
+    const prevConcepts = concepts.slice(0, index);
```

Comment:
nit: the prev/next thing seems a little verbose, maybe something like this?

```
if (r.expanded) {
  r.expanded = false;
  return;
}
concepts.forEach(c => c.expanded = false);
r.expanded = true;
```

Reply:
Done.